.github/workflows/conda-build.yml

```
@@ -5,57 +5,54 @@
+          channels: conda-forge,defaults,bioconda
+
+      - name: Create conda environment
+        run: conda env create -f environment.yml
+
+      - name: Activate conda environment
+        run: |
+          source $CONDA/etc/profile.d/conda.sh
+          conda activate sdrf-pipelines
+
+      - name: Install conda-build
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:8: Double quote to prevent globbing and word splitting

(shellcheck)